staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -147,3 +147,34 @@ func ComputeHistogramDelta(before, after model.Samples, label model.LabelName) {
 func makeKey(a, b model.LabelValue) string {
 	return string(a) + "___" + string(b)
 }
+
+// GetMetricValuesForLabel returns value of metric for a given dimension
+func GetMetricValuesForLabel(ms Metrics, metricName, label string) map[string]int64 {
+	result := make(map[string]int64)
+	samples, found := ms[metricName]
+	if !found {
+		return nil
```

Comment:
I don't know if this is important or not (probably not) but previous code returned an empty map in this case. If this does not matter, it's fine. Just saying.

Reply:
Done